lib/pavilion/resolver.py

```diff
@@ -439,12 +509,14 @@ def load(self, tests: List[str], host: str = None,
                     except TimeoutError:
                         pass
 
-        if outfile:
+        if outfile and not permuted_tests:
```

Comment:
The prints during test resolution are printed with `\r`, so the next percentage printed will overwrite the last. This prints a newline so the next printed thing doesn't overwrite that. 

So this is needed when `outfile and len(permuted_test) > 1`.